src/account/mixins/interface.cairo

```diff
@@ -0,0 +1,161 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.8.0 (account/mixins/interface.cairo)
+
+use starknet::ContractAddress;
+use starknet::account::Call;
+
+//
+// Mixins
+//
+
+#[starknet::interface]
```

Comment:
I understand the concept, but imo we shouldn't provide all the combinations with mixins. I lean towards having just one mixin for module, and if the user wants to accommodate then the approach should be to use the separated impls.

Reply:
>  imo we shouldn't provide all the combinations with mixins

That would make the proposal for having the mixin impl within the main component more feasible and it would also make the naming convention with mixins MUCH easier (like `AccountMixin` as opposed to `SRC6PubKeyDeclarerDeployer`). I was thinking that having all the options might make dx better, but I think I agree with one fully-loaded mixin impl per mod

Reply:
+1 to Eric, one mixin was the idea. I like how it's looking now